frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss

```diff
@@ -1,6 +1,6 @@
 .odc-resource-icon {
   font-size: 80%;
-  fill: var(--pf-global--info-color--200);
+  fill: #00b9e4;
```

Comment:
No pf4 variable for this?

Reply:
Nope, I couldn't find a global variable for this. I did find `$pf-color-light-blue-400` for this variable but I can't use it as we are not importing sass-utilities form patternfly anywhere in dev-console.

Reply:
Shouldn't there be a new rule for `.odc-resource-icon-statefulset` ?

Reply:
No, there wasn't any new rule for stateful. console team has this color as the default value and no color was mentioned for stateful